tf2onnx/convert.py

```diff
@@ -126,8 +123,7 @@ def main():
 
     model_proto = g.make_model("converted from {}".format(args.input))
```

Comment:
args.input here is just for "doc string".

Reply:
I have reverted my code.

Reply:
k. The line 
model_proto = g.make_model("converted from {}".format(args.input))
was always wrong, maybe since you are already changing in that area you could change it to:
model_proto = g.make_model("converted from {}".format(model_path))

Reply:
zhijiang has updated the path as suggested.
so i will going to merge this PR to unblock other PR.